File: pgm/ABC_Java/ABC_228_A.java
import java.io.*;
// import java.math.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final String[] arr = reader.readLine().split (" ");
        final String Yes = "Yes";
        final String No = "No";
        final int twentyFour = 24;
        final int S = Integer.parseInt (arr[0]);
        int T = Integer.parseInt (arr[1]);
        int X = Integer.parseInt (arr[2]);
        T = T < S ? T + twentyFour : T;
        X = X < S ? X + twentyFour : X;
        if (S <= X && X < T) {
            System.out.println (Yes);
        } else {
            System.out.println (No);
        }
        return;
    }
}

File: pgm/ABC_Kotlin/ABC_228_A.kt
// import java.io.BufferedOutputStream
// import java.io.BufferedReader
// import java.io.DataInputStream
// import java.io.FileInputStream
// import java.io.IOException
// import java.io.InputStreamReader
// import java.io.PrintWriter
// import java.lang.StringBuilder
// import java.time.LocalTime
// import java.time.format.DateTimeFormatter
// import java.util.Scanner
// import java.util.StringTokenizer

// import kotlin.math.*

const val Yes: String = "Yes"
const val No: String = "No"
const val space: String = " "
const val twentyFour: Int = 24

fun main() {
    var (S: Int, T: Int, X: Int) = readln().split(space).map { it.toInt() }
    T = if (T < S) T + twentyFour else T;
    X = if (X < S) X + twentyFour else X;
    if (S <= X && X < T) {
        println(Yes);
    } else {
        println(No);
    }
    kotlin.system.exitProcess(0)
}